contract/src/start-ibcToken.js

```diff
@@ -0,0 +1,190 @@
+/**
+ * @file core eval script* to start the ibcToken contract.
+ *
+ * * see test-gimix-proposal.js to make a script from this file.
+ *
+ * The `permit` export specifies the corresponding permit.
+ */
+// @ts-check
+
+import { E } from '@endo/far';
+import { makeMarshal } from '@endo/marshal';
```

Comment:
The gimix approach to core-eval proposals doesn't actually link imports. It erases import declarations, assuming they're for stuff like `E` that's provided by the core-eval compartment. So any calls to `makeMarshal` won't work.